docs/apps/authentik.md

```diff
@@ -39,7 +41,23 @@ sb install authentik
     1. Replace `saltbox_user` with the username you set when installing Saltbox.
     2. Replace `saltbox_password` with the password you set when installing Saltbox.
 
-### 3. Setup
+To enable Authentik with a single container, or on a per role basis, set the following [inventory](../saltbox/inventory/index.md) entry:
+
+``` yaml title="Authentik Single Container Toggle"
+
+radarr_traefik_sso_middleware: "authentik@docker"
+
+```
+
+To enable Authentik globally, set the following [inventory](../saltbox/inventory/index.md) entry:
+
+``` yaml title="Authentik Global Toggle"
+
+traefik_default_sso_middleware: "authentik@docker"
+
+```
+
+You would use one or the other, the global toggle will install Authentik together with Traefik when it is being installed, the role toggle requires manually installing Authentik. Traefik needs to be reinstalled at least once after updating to deploy the Authentik middleware to enable use of it. If you are using the global toggle, you will need to reinstall all of the other roles you use to enable Authentik.
```

Comment:
The global toggle will not result in the `authentik` role running when the `traefik` role runs. They will always need to be run as separate tags.

Reply:
It does actually :p

Reply:
https://github.com/saltyorg/Saltbox/blob/80d3c3e13b4538dd37330e3e0bc310c6e469de92/roles/traefik/tasks/main.yml#L90-L93

Reply:
For what its worth, I used saltys post in discord for that. It was copy and pasted from his announcement lmao

Reply:
I'm adding an oidc example to the docs though, so not quite done yet.